### ISSUE TITLE
night time weather icon

### ISSUE THREAD

**Issue by builderjer** [OpenVoiceOS/skill-ovos-weather#28]
I am creating this issue here so I don't forget.  The weather icon does not change from "sun" to "moon" after sunset.  The neon version does, and all of the icons are present, so it probably just needs to check the time against 'sunset' and 'sunrise'.

**Comment by emphasize** [comment]
maybe we can add a little twist. Even though it's nighttime i'd probably like to know the condition (cloudy, rain, etc.) 
if i can squeeze time, i'll check the options.

**Comment by builderjer** [comment]
Yes, all of the icons are there, just not added

**Comment by emphasize** [comment]
That involves work on both weather and homescreen skill btw.
Weather sends a code 0-7 and based on that homescreen is showing the svg.

I'm looking into overlay moon + condition a make them code 01, 02, ...

**Comment by builderjer** [comment]
neons weather skill works with all of the icons and the same homescreen as ovos

https://github.com/NeonGeckoCom/skill-weather

**Comment by emphasize** [comment]
Looking at the code it's relatively easy. `is_day` is passed to the Condition class. Just the map has to be expanded/broken down, overlays like [night cloudy](https://github.com/NeonGeckoCom/skill-weather/blob/dev/ui/images/partial_clouds_night.svg) to be created and the additional code sent to homescreen expanded.

**Comment by emphasize** [comment]
On a second look, not that easy [compat-wise](https://github.com/OpenVoiceOS/skill-ovos-weather/blob/9731dd4163eec6f40abdb7f9eacbd61b6b865d3c/weather_helpers/weather.py#L32).

Neon doesn't care about Mark1. And we are stuck because of that. I think @JarbasAl has to step in for suggestions.

**Comment by builderjer** [comment]
neon has their own weather skill

**Comment by JarbasAl** [comment]
> On a second look, not that easy [compat-wise](https://github.com/OpenVoiceOS/skill-ovos-weather/blob/9731dd4163eec6f40abdb7f9eacbd61b6b865d3c/weather_helpers/weather.py#L32).
> 
> Neon doesn't care about Mark1. And we are stuck because of that. I think @JarbasAl has to step in for suggestions.

dont worry about the Mk1 , it is not officially supported yet and the new codes should be handled there too!

**Comment by JarbasAl** [comment]
> neon has their own weather skill

not sure what the differences are, other than using the NeonMQ backend proxy instead of OpenMeteo as provider

but skills should be synced if Neon is diverging in anything other than provider, I assumed that was the case already with PRs being sent here

**Comment by builderjer** [comment]
Yes, I sent it here because neon already handled it, even if it is hacky
